simmechanics/data/icub3/ICUB_3_all_options.yaml

```diff
@@ -2,7 +2,7 @@
 robotName: iCub
 
 # Model base origin
-originXYZ: [0.0,0.0,0.80]
+originXYZ: [0.0,0.0,0.64]

```

Comment:
Can you add a line on this change in the PR description?

Reply:
I uniformed it with the `iCubGazeboV3`, the change on the height was done by @GiulioRomualdi (https://github.com/robotology/icub-models-generator/blame/b4d31861b14ea1d0cbc4e5952091633f708fdcc8/simmechanics/data/icub3/ICUB_3_all_options_gazebo.yaml.in#L5)